cli/DEVELOPMENT.md

```diff
@@ -0,0 +1,23 @@
+## Local development on the CLI
+
+The CLI is currently built by bundling the extension core and replacing the vscode rendering parts with a cli rendering engine. To _develop_ on the CLI you need to follow a few steps:
+
+1. Build the extension core from the root workspace folder by running `pnpm cli:bundle`
+
+2. Change into the cli folder `cd ./cli`
+
+3. Build & run the extension by running `pnpm start:dev`. If you want to use the CLI to work on its own code, you can run `pnpm start:dev -w ../` which will start it within the root workspace folder.
+
+4. While not required, it can be helpful to view log output in a separate terminal. To do this, open a new terminal window and run `pn logs`.
+
+## running tests
+
+`pn test` will test the cli specific code. Many changes require changing the extension code itself, which is tested from the root workspace folder.
+
+## Publishing to npm
+
+TODO: document how to publish versions to npm.
```

Comment:
```suggestion
- Merge the latest `Changeset version bump` PR to make sure that all changes are included in the CHANGELOG
- Wait for the `Build CLI Package` job to finish building on `main`
- When the job is done, an artifact will be attached to the action, download and extract the `.tgz` file inside of it
- Install the CLI through `npm install -g ./kilocode-cli-1.1.1.tgz`
- Make sure that the CLI works (Changes in the extension can sometimes conflict with the CLI):
  - Make the CLI increase a number in a JSON file
  - Ask the CLI to describe the `kilocode` project
- Run `npm publish` to publish the version after testing is complete
```

Reply:
I really don't like that option, as it changes the build machine between who publishes. If we use the GitHub action output we all know that we use the same build machine, and that there's no strange issues depending on the operating system or architecture.